samples/word/10-content-controls/insert-and-change-checkbox-content-control.yaml

```diff
@@ -0,0 +1,162 @@
+            paragraphs.load("$none"); // Don't need any properties; just start each paragraph with a content control.
+
+            await context.sync();
+
+            for (let i = 0; i < paragraphs.items.length; i++) {
+              let contentControl = paragraphs.items[i]
+                .getRange(Word.RangeLocation.start)
+                .insertContentControl(Word.ContentControlType.checkBox);
+            }
+            console.log("Checkbox content controls inserted: " + paragraphs.items.length);
+
+            await context.sync();
+          });
+        }
+
+        async function toggleCheckboxContentControls() {
+          // Toggles the isChecked property on checkbox content controls.
+          await Word.run(async (context) => {
+            let contentControls = context.document.getContentControls({
+              types: [Word.ContentControlType.checkBox]
+            });
+            contentControls.load("items");
+
+            await context.sync();
+
+            const length = contentControls.items.length;
+            console.log(`There are ${length} checkbox content controls.`);
+
+            if (length <= 0) {
+              return;
+            }
+
+            const checkboxContentControls = [];
+            for (let i = 0; i < length; i++) {
+              let contentControl = contentControls.items[i];
+              contentControl.load("id,checkboxContentControl/isChecked");
+              checkboxContentControls.push(contentControl);
+            }
+
+            await context.sync();
+
+            console.log("isChecked state before:");
+            const updatedCheckboxContentControls = [];
+            for (let i = 0; i < checkboxContentControls.length; i++) {
+              const currentCheckboxContentControl = checkboxContentControls[i];
+              const isChecked = currentCheckboxContentControl.checkboxContentControl.isChecked;
+              console.log(`id: ${currentCheckboxContentControl.id} ... isChecked: ${isChecked}`);
+
+              currentCheckboxContentControl.checkboxContentControl.isChecked = !isChecked;
+              currentCheckboxContentControl.load("id,checkboxContentControl/isChecked");
+              updatedCheckboxContentControls.push(currentCheckboxContentControl);
+            }
+
+            await context.sync();
+
+            console.log("isChecked state after:");
+            for (let i = 0; i < updatedCheckboxContentControls.length; i++) {
+              const currentCheckboxContentControl = updatedCheckboxContentControls[i];
+              console.log(
+                `id: ${currentCheckboxContentControl.id} ... isChecked: ${currentCheckboxContentControl.checkboxContentControl.isChecked}`
+              );
+            }
+          });
+        }
+
+        async function setup() {
+          await Word.run(async (context) => {
+            const body = context.document.body;
+            body.clear();
+            body.insertParagraph("One more paragraph. ", "Start");
```

Comment:
Can we use text like "Task 1, Task 2, etc" instead of the usual boilerplate text? I think it'll speak to the feature usage more.